packages/jest-haste-map/src/index.js

```diff
@@ -39,6 +39,7 @@ type Options = {
   cacheDirectory?: string,
   console?: Console,
   extensions: Array<string>,
+  forceNativeFind?: boolean,
```

Comment:
What I'm trying to say, in other words, is that this option forces using an API other than the file system one - it forces the use of the Child Process API via which it calls a system command. Yeah, it's not easy to explain that in an option name :)